hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -971,101 +976,103 @@ def enabled?(table_name)
     end
 
     #----------------------------------------------------------------------------------------------
-    # Return a new HColumnDescriptor made of passed args
-    def hcd(arg, htd)
+    # Return a new ColumnFamilyDescriptor made of passed args
+    def hcd(arg, tdb)
```

Comment:
Since this no longer returns `HColumnDescriptor` maybe `cfd` would be a better name?

Reply:
even if we do not change this name we will need to release note this change because it is visible to shell users but the return type changed.

Reply:
I agree. I went ahead and renamed `hcd` to `cfd` and added the removal of `hcd` to the custom "Release Note" field in Jira.